trollmoves/dispatcher.py

```diff
@@ -390,14 +390,17 @@ def create_dest_url(self, msg, client, conf):
         """Create the destination URL and the connection parameters."""
         defaults = self.config[client].copy()
         _verify_filepattern(defaults, msg)
-        connection_parameters = conf.get('connection_parameters', defaults.get('connection_parameters'))
-        host = conf.get('host', defaults['host'])
+        conf_with_defaults = defaults.copy()
```

Comment:
I think this copy is unnecessary. Yes, it makes the naming more clear, but still unnecessary copy since `defaults` is not used after this.

Reply:
Ok, would you be ok with this?
```suggestion
        conf_with_defaults = defaults
```

Reply:
Yes, or rename `defaults` to something more generic as `configuration` so the extra variable isn't needed.